### ISSUE TITLE
Atom tries to Start julia, but it stalls while displaying "Starting Julia..." In the julia-Console

### ISSUE THREAD

**Issue by ueliwechsler** [JunoLab/atom-julia-client#468]
I have the following problem:
Whenever I try to start julia in the console provided by uber-juno the programme stalls and nothing happens. 
The console displays the following message:
![unbenannt](https://user-images.githubusercontent.com/12156892/40217244-3c1c4a2c-5a5b-11e8-901d-9bacab0aba17.PNG)

It looks like the julia.exe can be found but it cannot be started or something keeps julia from executing properly.
Did anyone experience a similar behaviour? Any idea why this happens? Some days ago, I could install atom and the julia-packages without a problem on a similar machine.

The following packages / libs are installed:
Atom: uber-juno (and its dependencies), todo-show
julia: I added only the Atom packages.
I have installd, the Atom packages over the apm due to problems with self signed certificate in certificate chain.

Thanks for any help or inputs.

**Comment by pfitzseb** [comment]
Can you try to uncheck the `Enable Powershell Wrapper` option in the julia-client settings? Would also be good to make sure that Atom.jl loads fine, so try `using Atom` in a Julia REPL outside of Atom.

**Comment by ueliwechsler** [comment]
I did both. I have still the same problem.

**Comment by pfitzseb** [comment]
Alright, can you try opening the Atom devtools with `Ctrl-Shift-I` and take a look at the console? Are any errors reported there?

**Comment by ueliwechsler** [comment]
The following error message appeared when I tried to start Julia.
![image](https://user-images.githubusercontent.com/12156892/40466169-9a729888-5f24-11e8-94fb-fc531a995316.png)

**Comment by pfitzseb** [comment]
Can you navigate to the path mentioned in the error message and make sure `winpty-agent.exe` exists there? Please also check whether julia-client is correctly installed -- go to `Settings` -> `Packages` and search for julia-client; if there's _no_ settings button on that package something went wrong with the installation.

**Comment by ueliwechsler** [comment]
The latest tip did it for me. I checked the` winpty-agent.exe` file and I couldn't run it, because I have no permission to open .exe files in the \users\ subfolder (due to network security guidelines). So what I had to do was:
- copy the entire `\.atom\ `folder to a` C:\randomfolder\ `(where I have permission to run .exe)
- Add ` ATOM_HOME = C:\randomfolder\.atom` to my environment variables

Now, I can use the REPL in Atom. Thanks for your help.
